src/const.ts

```diff
@@ -34,4 +34,5 @@ export const DEFAULT_DECIMALS = 4
 export const DEFAULT_PRECISION = 18
 export const ELLIPSIS = '...'
 // TODO change infuraID for production
-export const INITIAL_INFURA_ENDPOINT = 'wss://rinkeby.infura.io/ws/v3/8b4d9b4306294d2e92e0775ff1075066'
+export const INFURA_ID = '8b4d9b4306294d2e92e0775ff1075066'
```

Comment:
At some point we should add a `process.env.INFURA_ID || '8b4d9b4306294d2e92e0775ff1075066' `